### PR TITLE
Fix Learn Chef tutorial link

### PR DESCRIPTION
**Pull request by tpetchel**
- We recently refactored a few things, updating a link to match.

Signed-off-by: tpetchel <tpetchel@gmail.com>

### CODE REVIEW

**Comment by chris-rock** [comment]
@tpetchel Thank you for updating the docs. Could you please sign-off your PR?

**Comment by tpetchel** [comment]
Hi @chris-rock - I already amended my commit (oops - I submitted my PR through GitHub and neglected that fact), but it looks like it didn't take. Let me try again.

**Comment by tpetchel** [comment]
I can't seem to amend the commit. Let me start over.
